### DISCUSSION TITLE
How to make payment with custom card (Native payment)

### DISCUSSION THREAD

**Discussion by Gursewak-Uppal** [flutter-stripe/flutter_stripe#296] [Q&A]
I want to use Stripe in my Flutter application. I don't have experience with Stripe. I used [flutter_credit_card](https://pub.dev/packages/flutter_credit_card) for getting the card details and after that I copied the details using CardDetails class like this.
```
 CardDetails _card = CardDetails();
 _card = _card.copyWith(
                                  number: "4242424242424242",
                                  expirationYear: int.tryParse("24"),
                                  expirationMonth: int.tryParse("12"),
                                  cvc: "453"); 
```

After that I tried to implement handle pay button functionality but I'm not able to understand how it works, Please anyone guide me step by step payment process, if possible.  I just want to test with PaymentIntent in mobile app side.

**Comment by jonasbark** [comment]
Did you check the examples? https://github.com/flutter-stripe/flutter_stripe/blob/main/example/lib/screens/custom_card_payment_screen.dart

**Reply by Gursewak-Uppal** [reply]
@jonasbark  [custom_card_payment_screen.dart](https://github.com/flutter-stripe/flutter_stripe/blob/main/example/lib/screens/custom_card_payment_screen.dart) not working for me. What I can do changes on it to make it working for me?
